gtm/variable_template.tpl

```diff
@@ -49,6 +49,13 @@ ___TEMPLATE_PARAMETERS___
       }
     ]
   },
+  {
+    "type": "CHECKBOX",
+    "name": "hasItems",
+    "checkboxText": "add to data layer items",
+    "simpleValueType": true,
+    "displayName": "Getting the prediction data from the items in the data layer"
```

Comment:
Perhaps something like this: "Add the request data below to the datalayer items".

I'm thinking the language should be consistent where possible to minimise confusion.

Thanks

Reply:
Yes, it makes sense. You’re right.
I changed it to “Add the request data below to the datalayer items".
Thank you so much for your feedback!